src/site/assignments/country-catalogue/Assignment1/src/Asn1.java

```diff
@@ -0,0 +1,51 @@
+import java.io.IOException;
```

Comment:
file path -- why include `Assignment1` in the path? Seems `/country-catalogue/src` is sufficient and more flexible.

Reply:
The Assignment1 is the project folder containing src, resources, and test. That's the folder that gradle then compresses and downloads. 

Though, I may be misunderstanding your point.

Reply:
It _could_ be, but the current `country-catalogue` dir holds the actual page/rst doc and project to download. There is nothing stopping the project having a better name though. 

On that point, I believe there is an open issue to go through and rename all the assignments to have better names anyways in a similar way to the topics were renamed from topicX -> something-useful. I envision doing that in a PR that updates all at once though.

Reply:
I checked, there is no open issue, so maybe it was an issue in the other course, or maybe I am misremembering. 

Regardless, I'll make an issue as I did do the change for 161 and should do it here (change being "assignmentX" -> "somethingUseful").

Reply:
https://github.com/jameshughes89/cs102/issues/46  has been closed. A new would be needed for assignments specifically.

Reply:
Agreed